src/status_im2/contexts/wallet/account/tabs/style.cljs

```diff
@@ -0,0 +1,4 @@
+(ns status-im2.contexts.wallet.account.tabs.style)
+
```

Comment:
Maybe creating these three namespaces with only one style definition is too much?
I'd just write it inline or in a def inside the `view` namespace, also their view namespaces are small.